validation/validation.go

```diff
@@ -135,6 +135,17 @@ func validateParams(path, method string, op *spec.Operation) error {
 	return nil
 }
 
+func validateDefinitions(definitions map[string]spec.Schema) error {
+	for name, def := range definitions {
+		for _, subDef := range def.Properties {
+			if len(subDef.Type) == 1 && subDef.Type[0] == "object" {
+				return fmt.Errorf("%s cannot have nested object types", name)
```

Comment:
I'd add a comment here describing why we can't have these, ideally linking to a go-swagger issue if it exists